manifests/params.pp

```diff
@@ -123,14 +125,21 @@
     }
 
     'Windows' : {
-      $package_ensure   = '7.1.0'
+      if $filebeat::package_provider == 'chocolatey' {
```

Comment:
It looks like tests are failing here since the scope is odd (filebeat:: vs local). Could you take a look at the tests?

Once everything is passing and you remove the WIP flag I'll get this merged.